### ISSUE TITLE
[TODO] Dataset setup & supervised training & HPO

### ISSUE THREAD

**Issue by jang1suh** [SNU-DRL/Attribution-ECG#1]
*** 추후 이 repository는 공개할 예정이니 다른 코드를 가져올 경우 출처를 코드에 명시해 주세요! 그리고 사용법도 README.md에 기록하며 작성 부탁드립니다. ***

1. Dataset setup
- Dataset: Icentia11K
- Task: beat classification
- Window size: frame size - 2049. approx. 8 sec.
- Label of a window: one of N(normal), S(PAC), V(PVC)
- Dataloader에서 받아야 할 추가 정보: window 안에서의 beat label 전부와 대응하는 위치
- Training - balanced, test - balanced.
- training=6000, test = 6000 (following Icentia11K paper)
- 12000개 sampling -> 10 different random seeds split / training.
- apply standardization, no augmentation

2. Supervised training
- Model: ResNet18, ResNet34에서 kernel을 1d로 바꾸고 kernel size를 늘린 것들 중 하나 선택
- Metric: (Balanced) Accuracy (Icentia11K 논문에서의 수치 참고)

3. HPO
- batch size
- learning rate
- epochs
